### PR TITLE
Create example.md

### PR DESCRIPTION
**Pull request by chainbelow**
Developers can test the cicero code while they build the node library

### CODE REVIEW

File: example.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+Name of the person to greet: "Fred Blogs".
+Thank you!
+</code>
+</pre>
+
+Now, change over to this directory, cd [install dir]/samples
+
+<pre>
+<code>
+$ node ../packages/cicero-cli/index.js parse --template . --dsl . --sample ./sample.txt
```
Comment (mttrbrts) [inline example.md]: We recommend running `npm install -g @accordproject/cicero-cli` to install the Cicero command-line tool globally. 

This line can then be simplified to `cicero parse`

**Comment by mttrbrts** [comment]
Hi @chainbelow, thank you for your contribution! We love to see new contributors ⭐️ 

We recommend that new users use the `yo` generator to create new templates, please see the guide here, https://docs.accordproject.org/docs/basic-create.html

By using the `yo` generator helps to reduce the number of manual steps and keeps the default template structure up to date with the latest developements in Cicero.

Does this address the problem that you're trying to solve? Instead, perhaps we should make the README file better to point people to relevant sections of the documentation?

Also for future reference, we require all contributors to sign each commit with a 'Developer Certificate of Origin`, there's further information in https://github.com/accordproject/cicero/blob/master/CONTRIBUTING.md

File: example.md (hunk 1 above)
Comment (chainbelow) [reply example.md]: The 'npm install ..' would be relevant to users, not software developers. Example: Folks may be extending or changing the cicero-cli package. That's the reason, we thought of it. Easier to test the changes. Just a suggestion. Perhaps example.md may not be the appropriate name, call it sandbox.md.

**Comment by chainbelow** [comment]
Hi. The yeoman generator again is relevant for users, not developers. Folks could be adding or changing the folder structure, like adding an IAM/ACL folder or other deployment folders. In our sandbox, we found this a better way.

**Comment by mttrbrts** [comment]
Thanks for clarifying @chainbelow, I understand now.  I hadn't appreciated that this comment was specific to developers / contributors.

Instead, I suggest that this content is best distributed as follows:

- The guidance on folder structure should go here, we describe it, but we aren't explicit about the folders, https://github.com/accordproject/techdocs/blob/master/docs/spec-template.md, which gets built to https://docs.accordproject.org/docs/spec-template.html  
- Then perhaps the rest could go either at https://github.com/accordproject/cicero/blob/master/packages/cicero-cli/README.md or https://github.com/accordproject/cicero/blob/master/DEVELOPERS.md

We could then give much better sign-posting from https://github.com/accordproject/cicero/blob/master/README.md?

**Comment by chainbelow** [comment]
Sounds good Matt. We will make the changes and send it across. Thanks.

**Pull request closed without merging**